src/Components/Form/FormFields/AutoExpandingTextInputFormField.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useRef } from "react";
+import TextAreaFormField, { TextAreaFormFieldProps } from "./TextAreaFormField";
+
+type AutoExpandingTextInputFormFieldProps = TextAreaFormFieldProps & {
+  id: string;
```

Comment:
why is id still being in props?

Reply:
should i make id optional or remove completely?